json-endpoints/jetpack/class.jetpack-json-api-get-comment-backup-endpoint.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+class Jetpack_JSON_API_Get_Comment_Backup_Endpoint extends Jetpack_JSON_API_Endpoint {
+	// /sites/%s/comments/%d/backup      -> $blog_id, $comment_id
+
+	protected $needed_capabilities = array(); // This endpoint is only accessible using a site token
+	protected $comment_id;
+
+	function validate_input( $comment_id ) {
+		if ( empty( $comment_id ) || ! is_numeric( $comment_id ) ) {
+			return new WP_Error( 400, __( 'You must specify a Comment ID', 'jetpack' ) );
+		}
+
+		$this->comment_id = intval( $comment_id );
+
+		return true;
+	}
+
+	protected function result() {
+		$comment = get_comment( $this->comment_id );
+		if ( empty( $comment ) ) {
+			return new WP_Error( 404, __( 'Comment not found', 'jetpack' ) );
+		}			
+		
+		return array(
+			'comment' => $comment->to_array(),
```

Comment:
fixed in 7c8607e. Now it looks more like this: 

<img width="1243" alt="postman" src="https://cloud.githubusercontent.com/assets/1369626/24395614/faec9072-13eb-11e7-828d-dc8a1fd9a75c.png">